server/views/pages/overview/partials/educationAndTrainingTabContents.njk

```diff
@@ -3,6 +3,30 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
 
+      <table class="govuk-table" id="functional-skills-table">
+        <caption class="govuk-table__caption govuk-table__caption--m">Functional skills</caption>
+
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Subject</th>
+            <th scope="col" class="govuk-table__header">Assessed on</th>
+            <th scope="col" class="govuk-table__header">Level</th>
+            <th scope="col" class="govuk-table__header">Type</th>
+          </tr>
+        </thead>
+
+        <tbody class="govuk-table__body">
+        {% for assessment in functionalSkills.assessments %}
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">{{ assessment.type | formatFunctionalSkillType }}</td>
+            <td class="govuk-table__cell">{{ assessment.assessmentDate | formatDate('D MMMM YYYY') }}</td>
+            <td class="govuk-table__cell">{{ assessment.grade }}</td>
+            <td class="govuk-table__cell">Induction</td>
```

Comment:
I have hardcoded the word "Induction" here, as at the moment all functional skills are from the prisoner's CIAG induction